ecl/resource_ecl_network_static_route_v2.go

```diff
@@ -29,78 +31,87 @@ func resourceNetworkStaticRouteV2() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			"region": &schema.Schema{
+			"region": {
 				Type:       schema.TypeString,
 				Optional:   true,
 				ForceNew:   true,
 				Computed:   true,
-				Deprecated: "This attribute is not used to set up the resource.",
+				Deprecated: "This attribute is not used to set up the resource",
 			},
-			"aws_gw_id": &schema.Schema{
+			"aws_gw_id": {
 				Type:          schema.TypeString,
 				Optional:      true,
 				ForceNew:      true,
-				ConflictsWith: []string{"azure_gw_id", "gcp_gw_id", "interdc_gw_id", "internet_gw_id", "vpn_gw_id"},
+				ConflictsWith: []string{"azure_gw_id", "fic_gw_id", "gcp_gw_id", "interdc_gw_id", "internet_gw_id", "vpn_gw_id"},
 			},
-			"azure_gw_id": &schema.Schema{
+			"azure_gw_id": {
 				Type:          schema.TypeString,
 				Optional:      true,
 				ForceNew:      true,
-				ConflictsWith: []string{"aws_gw_id", "gcp_gw_id", "interdc_gw_id", "internet_gw_id", "vpn_gw_id"},
+				ConflictsWith: []string{"aws_gw_id", "fic_gw_id", "gcp_gw_id", "interdc_gw_id", "internet_gw_id", "vpn_gw_id"},
 			},
-			"description": &schema.Schema{
+			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
-			"destination": &schema.Schema{
+			"destination": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
```

Comment:
terraform sdkのバリデーション関数が使用できそうです。サブネットマスクのサイズに制限がある場合は適宜修正してください。
```suggestion
				ForceNew: true,
				ValidateFunc: validation.CIDRNetwork(1, 32),
```